lib/cext/ruby.h

```diff
@@ -70,6 +70,10 @@ extern "C" {
 
 #define HAVE_RB_IO_T 1
 
+#ifdef RUBY_EXTCONF_H
+#include RUBY_EXTCONF_H
+#endif
```

Comment:
Used to compile a number of tests that add a function via `extconf.h`:
https://github.com/graalvm/truffleruby/blob/ec915bdcf1cccba94d60fd31c5f4b12d52c83919/test/mri/tests/cext/c/hash/extconf.rb#L7